pkg/storaged/dialog.jsx

```diff
@@ -471,32 +463,38 @@ export const dialog_open = (def) => {
 /* GENERIC FIELD TYPES
  */
 
-export const TextInput = (tag, title, options) => {
+export const TextInput = (tag, title, options, validated) => {
```

Comment:
I don't understand the addition of `validated` here.  I don't think any caller actually passes in a value for it, and I dount that the PF4 TextInput has a `validated` property.

Reply:
Oh it does https://www.patternfly.org/v4/components/text-input#props

And I pass it to the children here https://github.com/cockpit-project/cockpit/pull/15261/files/7830bd747feb82212dad66d45976d56407b5c2d0#diff-5b932241940b997840c4960d1bda3e884d3ba162f9aa007b761920aa61fdd245R257

Reply:
> Oh it does https://www.patternfly.org/v4/components/text-input#props

Ohh!  Once again I wasn't able to find any proper documentation. Google send me here: https://www.patternfly.org/2020.04/documentation/react/components/textinput which has no "props" section.

Reply:
> And I pass it to the children here

I think that doesn't actually go through the `TextInput` function, and will directly work on the result of the `render` function returned by `TextInput`.  I'll check this out more carefully.

(The things like `TextInput` are not React components, but I realize they look awefuly close.)

Reply:
I have pushed a FIXUP with how I think it needs to be done.  In the case of `SelectOne`, the React.clone approach would stick the "validated" prop on the `div` instead of on the `FormSelect` where it needs to be.